profiles/common/modules/features/splash_screen/splash_screen.module

```diff
@@ -76,6 +76,18 @@ function splash_screen_theme($existing, $type, $theme, $path) {
  * Implements hook_preprocess().
  */
 function splash_screen_preprocess_splash(&$variables) {
+  // Check the configuration.
+  if (variable_get('splash_screen_set_title', 0)) {
+    $site_name_value = variable_get('splash_screen_title_value');
+    $custom_splash_title = $site_name_value ? $site_name_value : FALSE;
```

Comment:
We can simplify by 

``$custom_splash_title = variable_get ('splash_screen_title_value',FALSE)``